### ISSUE TITLE
RN compatibility

### ISSUE THREAD

**Issue by Jahans3** [evgenyrodionov/redux-logger#254]
Hi guys, thanks for the good work on this.

I've been running this on RN projects since January and it's been working great, however I've started to notice slowdowns caused by logging. I understand logging isn't perfect in RN, but this package in particular has (seemingly) been slowing down apps and logging to the point where it becomes unusable.

Two examples:

After multiple reloads, logging slows down to the point where I have to wait 30+ seconds to see my logs, and then they all come at once. Killing the packager and rebooting fixes this. Perhaps there is a memory leak somewhere?

On another app, the logging appeared to be blocking actions from firing. I would send a thunk action, the initial action and promise inside would fire fine, but the action being sent from the promise resolving gets blocked. Weirdly, the actions would be released when there is user interaction.

Both of these issues disappeared when replacing redux-logger with redux-devtools. I have seen a similar issue that was raised and closed last year, but blocking the actions being fired made me think this library could have been responsible in some way.

Thanks

**Comment by evgenyrodionov** [comment]
Yeah, it’s our old issue #32. 

TLDR: console.log is really slow in RN and we can’t do anything with it.
